pkg/chain/upkeep_key.go

```diff
@@ -0,0 +1,32 @@
+package chain
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/smartcontractkit/ocr2keepers/pkg/types"
+	"strings"
+)
+
+type UpkeepKey []byte
```

Comment:
I would have struct here with block number and upkeep ID fields. It would be a more accurate implementation IMO.

Reply:
I totally agree, but to me that's out of the scope of this ticket? For this change alone I just wanted to swap out all existing references to the old UpkeepKey type with references to the new UpkeepKey interface

Reply:
I would not merge it with the bytes slice type so the ticket scope could be extended or the parent branch could be created until we have a struct here. Would it be difficult to implement it within this PR?

Reply:
I don't think it would be difficult to implement, it's just that, it was already a byte slice prior to this PR, and I wanted to have the least amount of changes necessary to get the interface changes over the line? So, I want to keep things as-is, without introducing too many changes in one PR, does that make sense?

Reply:
It makes sense to me but I would really want to change it to struct in the close future